src/app/datagrid/selection-row-mode/selection-row-mode.html

```diff
@@ -64,7 +64,7 @@ <h3>Multi selection</h3>
   <clr-dg-column>Creation date</clr-dg-column>
   <clr-dg-column>Favorite color</clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
+  <clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user" [clrDgRowAriaLabel]="'Select row for ' + user.name">
```

Comment:
This updates the demos on the angular docs application to create unique user names.